src/bioregistry/data/bioregistry.json

```diff
@@ -16351,6 +16351,23 @@
     ],
     "uri_format": "http://purl.obolibrary.org/obo/CIDO_$1"
   },
+  "ciliogenics": {
+    "contact": {
+      "email": "mustafapir29@gmail.com",
+      "name": "Mustafa S. Pir"
+    },
+    "contributor": {
+      "email": "shah.tanay2@northeastern.edu",
+      "github": "tanayshah2",
+      "name": "Tanay Shah",
+      "orcid": "0009-0001-1912-5132"
+    },
+    "description": "CilioGenics is an integrated and open source, community friendly database for ciliary genes.",
+    "example": "ENSG00000096872",
```

Comment:
This looks like an Ensembl gene ID. Does `ciliogenics` have its own identifier space or is it just resolving Ensembl IDs? If it's the latter, then this could instead be curated as a provider.

Reply:
Thanks, I can see that ciliogenics is now listed as a provider for ensembl but I think unless it has its own IDs, we would not list it as a separate entry at all. So if that's the case, we can delete this entry.